core/.changelog.d/4128.added

```diff
@@ -0,0 +1 @@
+Added translation of hard-coded 'Enable labeling' string
```

Comment:
this should be section "fixed" and the changelog entry is "Fix translation of the 'Enable labeling' screen."

Reply:
https://github.com/trezor/trezor-firmware/pull/4128/commits/014ca79f3c7289675389a26ed2b8037d1b2e7986